cpp/ql/test/library-tests/deduction_guides/test.ql

```diff
@@ -0,0 +1,4 @@
+import cpp
+
+from DeductionGuide d
+select d, d.getTemplateClass()
```

Comment:
Can we change this to:

```
from DeductionGuide d, string tc
where if exists(d.getTemplateClass()) then tc = d.getTemplateClass().toString() else tc = "<none>"
select d, d.getTemplateClass()
```

I think that would help clarify whether various deduction guides are themselves missing, or they don't have a template class associated.

Reply:
Added a second query instead of changing the existing one.